pyproject.toml

```diff
@@ -12,12 +12,15 @@ packages = [
     { include = "evergreen", from = "src" },
 ]
 
+[tool.poetry.requires-plugins]
+poetry-plugin-export = ">=1.8"
```

Comment:
Needed to be able to export runtime dependencies as a requirements.txt file for the GHA to be able to install deps for doc building.